drivers/overlay/overlay.go

```diff
@@ -1159,7 +1159,7 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 		}
 		mountData = label.FormatMountLabel(opts, options.MountLabel)
 		if len(mountData) > pageSize {
-			return "", fmt.Errorf("cannot mount layer, mount label too large %d", len(mountData))
+			return "", fmt.Errorf("cannot mount layer, mount label %s too large %d", options.MountLabel, len(mountData))
```

Comment:
If I were seeing this error as a user my first question would be, "what is the limit?" Is this a likely error condition? If so, it might be polite to include something like "length (%d) must be <= %d", ..., len(mountData), pageSize)

Reply:
This is a very unlikely message.